src/codemagic/apple/resources/beta_build_localization.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Optional
+
+from .resource import Relationship
+from .resource import Resource
+
+
+class BetaBuildLocalization(Resource):
+    """
+    https://developer.apple.com/documentation/appstoreconnectapi/betabuildlocalization
+    """
+
+    @dataclass
+    class Attributes(Resource.Attributes):
+        locale: str
+        whatsNew: Optional[str] = None
```

Comment:
I had somehow missed that locale is defined here with type `str`, while there is a dedicated enumeration available for it.
```suggestion
    class Attributes(Resource.Attributes):
        locale: Locale
        whatsNew: Optional[str] = None

        def __post_init__(self):
            if isinstance(self.locale, str):
                self.locale = Locale(self.locale)
```